src/main.c

```diff
@@ -41,6 +43,15 @@ on_name_lost (gpointer instance G_GNUC_UNUSED, gpointer loop)
 int
 main (int argc G_GNUC_UNUSED, char ** argv G_GNUC_UNUSED)
 {
+  gboolean bLocal = rda_session_is_local ();
+
+  if (!bLocal)
+  {
+      g_message ("We are in a remote session, exiting.");
```

Comment:
If we exit the indicator and we launch it as a systemd service, won't it immediately come back? Wouldn't it make more sense to float around in background and be invisible to the user?

Reply:
> If we exit the indicator and we launch it as a systemd service, won't it immediately come back? Wouldn't it make more sense to float around in background and be invisible to the user?

Ooops, you're right. I was testing it with XDG autostart, totally forgot about systemd. It won't be very elegant, but we'll have to zombify it.